travis.sh

```diff
@@ -54,29 +54,22 @@ rebuild() {
 
 # test druntime, phobos, dmd
 test() {
-    test_dub_package
     make -j$N -C ../druntime -f posix.mak MODEL=$MODEL unittest
     make -j$N -C ../phobos -f posix.mak MODEL=$MODEL unittest
     test_dmd
 }
 
 # test dmd
 test_dmd() {
+    local make_targets="run_tests test_dub"
```

Comment:
I didn't add `test_dub` to the default `all` target as it would then by run on the auto-tester as well. While we do want to test this on the long run, I wasn't sure whether `dub` is available on the auto-tester, so I left testing this open for another PR.